src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add(res = 0, num) {
+      result += num;
+
+      return calculator;
```

Comment:
I suggest you to return this instead of calculator as it will make your code more scalable  
```suggestion
    add(res = 0, num) {
      result += num;

      return this;
```